onnxruntime/python/onnxruntime_pybind_state_common.h

```diff
@@ -483,6 +483,8 @@ std::shared_ptr<IExecutionProviderFactory> CreateExecutionProviderFactory_Cuda(c
 std::shared_ptr<IExecutionProviderFactory> CreateExecutionProviderFactory_Dnnl(int use_arena);
 std::shared_ptr<IExecutionProviderFactory> CreateExecutionProviderFactory_OpenVINO(const OrtOpenVINOProviderOptions* params);
 std::shared_ptr<IExecutionProviderFactory> CreateExecutionProviderFactory_Nuphar(bool, const char*);
+std::shared_ptr<IExecutionProviderFactory> CreateExecutionProviderFactory_OpenCL();
+std::shared_ptr<IExecutionProviderFactory> CreateExecutionProviderFactory_OpenCL(bool use_fp16);
```

Comment:
Can we just have one entry here? The EPs that have multiple are taking different types in as params. This just seems to be a duplicate to avoid explicitly setting use_fp16.

Reply:
The former one is removed.